config/config.go

```diff
@@ -186,8 +186,13 @@ func setTxPool(ctx *cli.Context, cfg *evmcore.TxPoolConfig) error {
 	if ctx.GlobalIsSet(flags.TxPoolRejournalFlag.Name) {
 		cfg.Rejournal = ctx.GlobalDuration(flags.TxPoolRejournalFlag.Name)
 	}
+	// TxPoolPriceLimitFlag will be deprecated in the future.
+	// hence TxPoolMinTipFlag is favored.
 	if ctx.GlobalIsSet(flags.TxPoolPriceLimitFlag.Name) {
-		cfg.PriceLimit = ctx.GlobalUint64(flags.TxPoolPriceLimitFlag.Name)
+		cfg.MinimumTip = ctx.GlobalUint64(flags.TxPoolPriceLimitFlag.Name)
```

Comment:
Do you think we should log a warning here or at another place if this flag is used to inform the operator?

Reply:
warning log added